src/main/java/com/zendesk/maxwell/replay/FilePathParser.java

```diff
@@ -0,0 +1,103 @@
+package com.zendesk.maxwell.replay;
+
+import com.amazonaws.util.StringInputStream;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.StreamTokenizer;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Pattern;
+
+import static java.io.StreamTokenizer.TT_EOF;
+import static java.io.StreamTokenizer.TT_WORD;
+
+/**
+ * @author udyr@shlaji.com
+ */
+public class FilePathParser {
+	private final String path;
+	private StreamTokenizer tokenizer;
+
+	private FilePathParser(String path) {
+		this.path = path;
+	}
+
+	public static List<ReplayFilePattern> parse(String path) {
+		return new FilePathParser(path).parse();
+	}
+
+	private List<ReplayFilePattern> parse() {
+		try (InputStreamReader inputStream = new InputStreamReader(new StringInputStream(path))) {
+			this.tokenizer = new StreamTokenizer(inputStream);
+			return doParse(inputStream);
+		} catch (IOException e) {
+			throw new RuntimeException(e.getMessage(), e);
+		}
+	}
+
+	private List<ReplayFilePattern> doParse(InputStreamReader inputStream) throws IOException {
```

Comment:
uh, this is a weird copy and paste from the filter pattern tokenizer... but I don't think you need to do any of this.

Reply:
Yes, I referred to the Filter